modelplugin/aether-2.0.0/modelplugin/modelplugin.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021-present Open Networking Foundation.
```

Comment:
I think this file is not needed - but I was expecting to see a `generated.go`

Reply:
probably forgot to add it. Let me see if sdcore-adapter needs the modelplugin.go.